src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -807,7 +825,11 @@ def storeTemplate(self, positionNumber, charBufferNumber = 0x01):
 
         ## DEBUG: Template stored successful
         if ( receivedPacketPayload[0] == FINGERPRINT_OK ):
-            return True
+            ## Return index when positionNumber was -1
+            if ( found == True ):
```

Comment:
I know! The problem is users may already use the boolean return value in their programs so changing the return value from boolean to a tuple will crash their programs! Shall I do it regardless? Normally I would overload the function but this is not possible in Python...